charts/renku/values.yaml

```diff
@@ -314,6 +314,10 @@ jupyterhub:
     extraConfig: |
       import os
 
+      # JupyterHub uses GITLAB_HOST which is a misnomer -- we use GITLAB_URL
+      # everywhere so set this mapping here
+      os.environ['GITLAB_HOST'] = os.getenv('GITLAB_URL')
```

Comment:
I hope we solve it quickly in https://github.com/jupyterhub/oauthenticator/issues/191.